packages/components/src/text-input/tests/chromatic/SearchInput.chroma.jsx

```diff
@@ -0,0 +1,144 @@
+import { Inline, Stack } from "@components/layout";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong